lms/djangoapps/courseware/views.py

```diff
@@ -918,7 +919,7 @@ def course_about(request, course_id):
 def progress(request, course_id, student_id=None):
     """ Display the progress page. """
 
-    course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
+    course_key = CourseKey.from_string(course_id)
```

Comment:
Is this related to the CCX changes, or it is an unrelated but useful improvement?